lib/simple-db.js

```diff
@@ -0,0 +1,173 @@
+/**
```

Comment:
I'll fix it up here. I've used this code elsewhere and have some confidence that it works as expected, so I'd rather stick with it rather than pulling in an IDB Promise wrapper from `npm`. If we do refactor this out to be a standalone library it should be a quick PR in the future.